src/main.ts

```diff
@@ -49,10 +49,6 @@ async function run(): Promise<void> {
         logger.info('Start getting job info...')
         const jobInfo = await getJobInfo(octokit)
         logger.debug(`jobInfo: ${jobInfo.id}, ${jobInfo.name}`)
-        if (!jobInfo.id || !jobInfo.name) {
```

Comment:
I think, if job info is empty (`{}`), which is the case that there is no resource access error but couldn't get job info even retried `10` times , we should notice and return here.